lib/countly-bulk-user.js

```diff
@@ -171,22 +171,29 @@ function CountlyBulkUser(conf) {
     /**
     * Remove consent for specific feature, meaning, user opted out to track that data (either core feature of from custom feature group)
     * @param {string|array} feature - name of the feature, possible values, "sessions","events","views","crashes","attribution","users" or customly provided through {@link CountlyBulkUser.group_features}
+    * @param {Boolean} enforceConsentUpdate - regulates if a request will be sent to the server or not. If true, removing consents will send a request to the server and if false, consents will be removed without a request 
     */
     this.remove_consent = function(feature) {
         log("Removing consent for " + feature);
-        if (feature.constructor === Array) {
+        this.remove_consent_internal(feature, true);
+    }    
+    //removes consent without updating
+    this.remove_consent_internal = function(feature, enforceConsentUpdate) {
+        //if true updateConsent will execute when possible
+        enforceConsentUpdate = enforceConsentUpdate || false; 
+            if (feature.constructor === Array) {
```

Comment:
Incorrect indentation, code should be eslinted
and we should add a check for it through github actions for PRs